src/main/java/Model/Person.java

```diff
@@ -0,0 +1,46 @@
+package Model;
+
+import javax.persistence.*;
+
+@Entity
+public abstract class Person {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Integer id = null;
+    private String name = null;
+    private String type = null;
+
+    public Person() {}
+
+    public Person(String name){
+        this.name = name;
+    }
+    public Person(String name, String type){
+        this.name = name;
+        this.type = type;
```

Comment:
what is the type? Just student or instructor?

Reply:
yes, could make it an enum

Reply:
this constructor is for the Student and Instructor classes, this constructor is called in their constructors.